xml/obs_admin_components.xml

```diff
@@ -7,8 +7,147 @@
   %entities;
 ]>
 <section id="cha.obs.admin.components">
- <title>Server Components</title>
+ <title>OBS Components</title>
  <para>
-  To be written...
+   The OBS is not a monolithic server, it consists of multiple daemons that
+   fulfill different tasks.
  </para>
+ <figure>
+  <title>OBS Component Overview</title>
+  <mediaobject>
+   <imageobject>
+    <imagedata fileref="obs_components.png" width="800" />
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <section>
+   <title>Frontend</title>
+   <para>
+     The OBS Frontend is a Ruby on Rails application that manages the access and
+     manipulation of the OBS data. It provides a web user interface and an
+     application programming interface to do so. Both can be used to create,
+     read update and delete users, projects/packages, requests and other
+     objects. It also implements additional sub-systems like authentication,
+     search or email notifications.
+   </para>
+ </section>
+ <section>
+   <title>Backend</title>
+   <para>
+     The OBS Backend is a collection of Perl applications that manage the source
+     files and build jobs of the OBS.
+   </para>
+   <section>
+     <title>Source Server</title>
+     <para>
+       Maintains the source repository and project/package configurations.
+       It is the only interface for the Frontend and it may forward requests to
+       one of the repository servers. Each OBS installation only has one
+       Source Server.
+     </para>
+   </section>
+   <section>
+     <title>Repository Server</title>
+     <para>
+       A repository server provides access to the repositories of
+       some projects/packages/architectures. It also maintains the
```

Comment:
The text is from the [backend docu(https://github.com/openSUSE/open-build-service/blob/master/src/backend/DESIGN) ;-)